5.4.kelim_shlovim.py

```diff
@@ -1,13 +1,13 @@
-# The function takes in an arbitrary number of lists as positional arguments and yields the elements of these lists
-# in an interleaved order
-def interleave(*lists):
-    index = 0
-    max_len = max(len(lst) for lst in lists)
-    while index < max_len:
-        for lst in lists:
-            if index < len(lst):
-                yield lst[index]
-        index += 1
-
-
-print(list(interleave('abc', [1, 2, 3], ('!', '@', '#'))))
+# The function takes in an arbitrary number of lists as positional arguments and yields the elements of these lists
+# in an interleaved order
+def interleave(*lists):

```

Comment:
Your function is semi-documented.
Check again how to document functions, where is the spot we leave our documentation and make sure to also document the parameters it receives and what it returns (if it yields then it should be mentioned explicitly)